### PR TITLE
Actor zeta@1.0.0a4

### PR DESCRIPTION
**Pull request by kotbegemot**

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -60,12 +55,16 @@ def build(self):
...
         self.copy(pattern="*.dll", dst="bin", keep_path=False)
         self.copy(pattern="*.lib", dst="lib", keep_path=False)
         self.copy(pattern="*.a", dst="lib", keep_path=False)
         self.copy(pattern="*.so*", dst="lib", keep_path=False)
         self.copy(pattern="*.dylib", dst="lib", keep_path=False)
 
     def package_info(self):
+        if self.settings.os == "Linux":
+            self.cpp_info.system_libs.append("pthread")
+        if self.settings.os == 'Windows':
```
Comment (theirix) [inline conanfile.py:68]: Deleting fPIC should be done in `config_options` not in the package_info

File: conanfile.py [hunk 2]
```diff
@@ -60,12 +55,16 @@ def build(self):
 
     def package(self):
```
Comment (theirix) [inline conanfile.py:56]: The license file from the upstream package should be installed:
```
self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
```

Comment (kotbegemot) [reply conanfile.py:57]: done

File: conanfile.py (hunk 1 above)
Comment (kotbegemot) [reply conanfile.py]: done

**Pull request merged**